app/views/users/_friendship.html.erb

```diff
@@ -0,0 +1,26 @@
+<% if @recieved.include?(user) && user.id != current_user.id %>
```

Comment:
**[REQUIRED]**
* Views related logic should be stored in helpers. ([Screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/rails/section/no_logic_in_helpers_requirement.png) from Pathwright.
Instead of calling `@recieved.include?(user)` in the views, we can create a helper method in 'app/helpers/application_helper.rb`
```ruby
# def like_or_dislike_btn(post)
def is_present?(list, record)
    list.include? record
end
```
Then just call the method in the views.
```ruby
is_present?(@recieved, user)
```
* [Where Do I Put My Code?](http://codefol.io/posts/Where-Do-I-Put-My-Code)
* [The Beginner's Guide to Rails Helpers](https://mixandgo.com/learn/the-beginners-guide-to-rails-helpers)